src/NuGet.Core/NuGet.Common/PathUtil/FileUtility.cs

```diff
@@ -42,7 +42,7 @@ public static async Task DeleteWithLock(string filePath)
                 {
                     Delete(filePath);
 
-                    return Task.FromResult(0);
+                    return TaskResult.Zero;
```

Comment:
I am curious why we can't use `Task.CompletedTask` here.

Reply:
This one actually returns a Task<int>. The others were returning Task.